types/node/index.d.ts

```diff
@@ -5821,3 +5821,735 @@ declare module "async_hooks" {
      */
     export function createHook(options: HookCallbacks): AsyncHook;
 }
+
+declare module "http2" {
+    import * as events from "events";
+    import * as fs from "fs";
+    import * as net from "net";
+    import * as stream from "stream";
+    import * as tls from "tls";
+    import * as url from "url";
+
+    export interface Headers {
+        [headerField: string]: string | string[];
+    }
```

Comment:
Sure thing, do you think (from a philosophical standpoint) that there might be a problem with importing from `"http"` though? `http2` doesn't depend on `http`.

Also, I noticed that outgoing headers have additional types in `http`; I'll see if the same is true in `http2`.